controllers/actions.github.com/resourcebuilder.go

```diff
@@ -133,7 +130,10 @@ func (b *resourceBuilder) newScaleSetListenerPod(autoscalingListener *v1alpha1.A
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      autoscalingListener.Name,
 			Namespace: autoscalingListener.Namespace,
-			Labels:    newLabels,
+			Labels: map[string]string{
+				"auto-scaling-runner-set-namespace": autoscalingListener.Spec.AutoscalingRunnerSetNamespace,
+				"auto-scaling-runner-set-name":      autoscalingListener.Spec.AutoscalingRunnerSetName,
```

Comment:
we have a key for `auto-scale-runner-set-name`, (`LabelKeyAutoScaleRunnerSetName`). we might need to make sure all labels match each other.

Reply:
I'll extract it
I saw in other parts that we are using string literal, so just made it like that, but great suggestion!